libwayshot/src/lib.rs

```diff
@@ -452,6 +452,8 @@ impl WayshotConnection {
             }
         };
 
+        let mut layer_shell_surfaces = Vec::new();
```

Comment:
We know the wl_output count right? Can we just make a static array to reduce allocations?

Reply:
That should be possible, I'll look into it.

Reply:
Actually, no, we don't know the wl_output count at compile time so I can't see how we could use a simple static array. What we can do is use Vec::with_capacity to init the vec with the number of output frames, is this what you meant?